classes/fields/pick.php

```diff
@@ -168,6 +168,28 @@ public function options() {
 				),
 				'dependency' => true,
 			),
+			static::$type . '_display_format_multi'   => array(
+				'label'      => __( 'Display Format', 'pods' ),
+				'help'       => __( 'Used as format for front-end display', 'pods' ),
+				'depends-on' => array( static::$type . '_format_type' => 'multi' ),
+				'default'    => 'default',
+				'type'       => 'pick',
+				'data'       => array(
+					'default' => __( 'Item, Item and Item', 'pods' ),
+					'custom'  => __( 'Custom separator', 'pods' ),
+				),
+				'dependency' => true,
+			),
+			static::$type . '_display_format_separator'   => array(
+				'label'      => __( 'Display Format Separator', 'pods' ),
+				'help'       => __( 'Used as separator for front-end display', 'pods' ),
+				'depends-on' => array(
+					static::$type . '_display_format_multi' => 'custom',
+					static::$type . '_format_type'          => 'multi',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'_format_type'" and double arrow, but found 10.

<sub>hash: a0381e6cc21f80bee0b044f528a2b89f</sub>